code/io/joy_ff-sdl.cpp

```diff
@@ -42,9 +52,6 @@ static haptic_effect_t pShootEffect;
 static haptic_effect_t pSecShootEffect;
 static haptic_effect_t pSpring;
 static haptic_effect_t pDock;
-static haptic_effect_t pDeathroll1;
-static haptic_effect_t pDeathroll2;
-static haptic_effect_t pExplode;
```

Comment:
Rather surprised at the removal of these effects, but I see how they are filled in by other effects with modifications, so I suppose that works.